migrations/1578567421515-addGranularSchoolEditPermissions.js

```diff
@@ -0,0 +1,88 @@
+const mongoose = require('mongoose');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'mongoose'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4605061)